DiceKit/Die.swift

```diff
@@ -116,3 +116,14 @@ extension Die: ExpressionType {
 public func == (lhs: Die, rhs: Die) -> Bool {
     return lhs.sides == rhs.sides
 }
+
+// MARK: - Debug
```

Comment:
Also, should be a new line after it. We only don't do newlines in the tests, because there we are naming the extension as well. If anything, we should use newlines there as well, just pointing out why there is a difference.